src/Dialect/Json.php

```diff
@@ -82,13 +82,16 @@ public function inspectJsonColumns()
             if (!$this->showJsonColumns) {
                 $this->hidden[] = $col;
             }
-            $obj = json_decode($this->$col);
 
-            if (is_object($obj)) {
-                foreach ($obj as $key => $value) {
-                    $this->flagJsonAttribute($key, $col);
-                    if ($this->showJsonAttributes) {
-                        $this->appends[] = $key;
+            if(isset($this->attributes[$col])) {
```

Comment:
I understand the change and agree in principle however this may break backward compatibility for users that are using magic methods to reference attributes that are computed... instead could we do something like this:
```php
if(isset($this->attributes[$col])) {
    $obj = json_decode($this->attributes[$col]);
}
else {
    $obj = json_decode($this->$col);
}

if (is_object($obj)) {
    foreach ($obj as $key => $value) {
        $this->flagJsonAttribute($key, $col);
        if ($this->showJsonAttributes) {
            $this->appends[] = $key;
        }
    }
}
```

Reply:
That doesn't quite work for me. We have numerous organizations which have the "configuration" = null in the database. In those cases, isset will return false and invoke the magic method. This revised version with array_key_exists should be a good compromise.